src/OpenCV_Mat.jl

```diff
@@ -195,25 +195,25 @@ function pixset(img, row::Int, col::Int, value)
   # RGB images (value::  std::vector<double>)
   elseif (cvtypeval(img) == CV_8SC3)
     cppvec = tostdvec(value)
```

Comment:
Not sure still whether the conversion to `std::vec` should be done in this function.
In any case, there is no need to repeat `cppvec = tostdvec(value)` in every elseif statement.
Just state it once as shown below.

```julia
function pixset(img, row::Int, col::Int, value)
  (row < 0 || col < 0 || row > rows(img) || col > cols(img)) ? throw(BoundsError()) : nothing
  cppvec = tostdvec(value)

. . . code not shown . . .

  elseif (cvtypeval(img) == CV_8SC3)
    @cxx at_vc(img, row, col, cppvec);
  elseif (cvtypeval(img) == CV_8UC3)
    @cxx at_v3b(img, row, col, cppvec);
  elseif (cvtypeval(img) == CV_16SC3)
    @cxx at_v3s(img, row, col, cppvec);
  elseif (cvtypeval(img) == CV_16UC3)
    @cxx at_v3us(img, row, col, cppvec);
  elseif (cvtypeval(img) == CV_32SC3)
    @cxx at_v3i(img, row, col, cppvec);
  elseif (cvtypeval(img) == CV_32FC3)
    @cxx at_v3f(img, row, col, cppvec);
  elseif (cvtypeval(img) == CV_64FC3)
    @cxx at_v3d(img, row, col, cppvec);
  else throw(ArgumentError("Image format not recognized!"))
  end
end
```